libraries/ms-drivers/src/ltc_afe_impl.c

```diff
@@ -211,13 +243,10 @@ StatusCode ltc_afe_impl_trigger_cell_conv(LtcAfeStorage *afe) {
 }
 
 StatusCode ltc_afe_impl_trigger_aux_conv(LtcAfeStorage *afe, uint8_t device_cell) {
-  // TODO(SOFT-9): Update GPIO usage to match updated design
-
-  // configure the mux to read from cell
-  // we use GPIO2, GPIO3, GPIO4, GPIO5 to select which input to read
-  // corresponding to the binary representation of the cell
+  // We use GPIO 1 to read data from the mux
   prv_write_config(afe, (device_cell << 4) | LTC6811_GPIO1_PD_OFF);
-
+  prv_mux_write_comm_register(afe, device_cell);
+  prv_mux_send_comm_register(afe);
```

Comment:
imo you might wanna prefix these with `prv_aux_` or `prv_aux_spi_` instead of `prv_mux_`. The only mux specific thing in this code is the content of the message

Reply:
@g-jessmuir can prolly comment better if this is a good idea, but we may want to generalize write_comm register to take 3 bytes and a number which indicates how many bytes you want to send. That way this code is entirely decoupled from the mux beyond this line.